src/pages/TermsOfService.test.ts

```diff
@@ -15,11 +15,33 @@ jest.mock('react-notifications-component', () => {
   };
 });
 
+type NavExports = typeof import('src/libs/nav');
+jest.mock(
+  'src/libs/nav',
+  (): NavExports => ({
+    ...jest.requireActual('src/libs/nav'),
+    goToPath: jest.fn(),
+  })
+);
+type AuthExports = typeof import('src/libs/auth');
+jest.mock(
+  'src/libs/auth',
+  (): AuthExports => ({
+    ...jest.requireActual('src/libs/auth'),
+    signOut: jest.fn(),
+  })
+);
+
 const setupMockAjax = (termsOfService) => {
   const getTos = jest.fn().mockReturnValue(Promise.resolve('some text'));
   const getTermsOfServiceComplianceStatus = jest.fn().mockReturnValue(Promise.resolve(termsOfService));
   const getStatus = jest.fn().mockReturnValue(Promise.resolve({}));
-  Ajax.mockImplementation(() => ({
+  const acceptTos = jest.fn().mockReturnValue(Promise.resolve({ enabled: true }));
+  const rejectTos = jest.fn().mockReturnValue(Promise.resolve({ enabled: true }));
+  const getFenceStatus = jest.fn();
+  const getNihStatus = jest.fn();
+
+  (Ajax as jest.Mock).mockImplementation(() => ({
```

Comment:
There's an `asMockedFn` helper in src/testing/test-utils that preserves the type signature of the mocked function.

```suggestion
  asMockedFn(Ajax).mockImplementation(() => ({
```